content/ko/docs/reference/glossary/workload.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 워크로드(Workloads)
+id: workloads
+date: 2019-02-13
+full_link: /docs/concepts/workloads/
+short_description: >
+   워크로드는 클러스터의 컨테이너를 동작시키고 관리하기 위해 사용하는 오브젝트이다.
+
+aka: 
+tags:
+- fundamental
+- core-object
+- workload
+---
+  워크로드는 클러스터의 컨테이너를 동작시키고 관리하기 위해 사용하는 오브젝트이다.
+
+<!--more--> 
+
+쿠버네티스는 
+애플리케이션의 현재 상태에 따라 워크로드의 디플로이먼트와 업데이트를 수행한다.
+워크로드는 데몬셋, 디플로이먼트, 잡, 파드, 레플리카셋, 레플리케이션컨트롤러, 스테이트풀셋과 같은 오브젝트를 포함한다.
+
+예를 들어, 웹 요소와 데이터베이스 요소가 있는 워크로드는 
+데이터베이스를 {{< glossary_tooltip text="파드" term_id="pod" >}}의 한 
+{{< glossary_tooltip text="스테이트풀셋" term_id="StatefulSet" >}} 안에서 실행할 것이며, 
```

Comment:
```suggestion
{{< glossary_tooltip text="스테이트풀셋" term_id="statefulset" >}} 안에서 실행할 것이며,
```

Reply:
@sftim, Thank you for your comments :)
I noticed the case mismatches. 

The documents in this PR are translations of original English documents. The case mismatches in the term ID are from the original English documents. Korean localization team tries to make translated doc similar with its original doc as possible. (sometimes, even a bug in original doc)

After the original docs are fixed in master branch, we update related Korean docs in Korean localization branch (kind of splint style) and merge the branch to master branch.

Since the case mismatches in the IDs do not occur problem in the built document (in preview, the tooltip works), I hope to remain this PR as is for this moment until the case mismatches are fixed in master branch.
Is this way ok to you? @sftim

Reply:
I must have done the wrong check - sorry! It's OK to treat this as resolved.